types/sharp-timer/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": [
+      "es6"
+    ],
+    "noImplicitAny": false,
```

Comment:
Can you please turn these strict flags (or at least `noImplicitAny`) to `true` and amend the PR?